its/src/test/java/com/sonar/it/shared/TestUtils.java

```diff
@@ -181,14 +187,22 @@ public static String replaceLtsVersion(String version) {
   }
 
   public static void reset(Orchestrator orchestrator) {
-    orchestrator.getServer().newHttpCall("api/orchestrator/reset")
-      .setMethod(HttpMethod.POST)
+    // We add one day to ensure that today's entries are deleted.
+    Instant instant = Instant.now().plus(1, ChronoUnit.DAYS);
+
+    // The expected format is yyyy-MM-dd.
+    String currentDateTime = DateTimeFormatter.ISO_LOCAL_DATE
+      .withZone( ZoneId.of("UTC"))
+      .format(instant);
+
+    LOG.info("TEST SETUP: deleting projects analyzed before: " + currentDateTime);
+
+    orchestrator.getServer()
+      .newHttpCall("/api/projects/bulk_delete")
       .setAdminCredentials()
+      .setMethod(HttpMethod.POST)
+      .setParams("analyzedBefore", currentDateTime)
```

Comment:
Java DateTime API is a daytime nightmare. Can you replace it with this and remove all the orchestration and imports?
```suggestion
      .setParams("qualifiers", "TRK")
```

Reply:
Currently the API documentation says: `At least one parameter is required among analyzedBefore, projects and q` and this is a bit misleading.

Reply:
Are you sure about this? The API returns now 400. See: https://sonarsource.visualstudio.com/DotNetTeam%20Project/_build/results?buildId=22472&view=logs&j=5a0b2a7a-818d-5a4c-a639-e2568d087f6b&t=407cbd3b-4ba1-5b9c-259b-e19d650c77d2

Reply:
No. There's not-so-well-named method `checkAtLeastOneParameterIsPresent` that validates the request on SQ side :(